src/main/scala/nl/biopet/tools/vepnormalizer/ArgsParser.scala

```diff
@@ -5,10 +5,31 @@ import java.io.File
 import nl.biopet.utils.tool.AbstractOptParser
 
 class ArgsParser(cmdName: String) extends AbstractOptParser[Args](cmdName) {
-  opt[File]("inputFile")
-    .abbr("i")
-    .unbounded()
-    .required()
-    .maxOccurs(1)
-    .action((x, c) => c.copy(inputFile = x))
+  head(s"""|$cmdName - Parse VEP-annotated VCF to standard VCF format """)
+
+  opt[File]('I', "InputFile") required () valueName "<vcf>" action { (x, c) =>
+    c.copy(inputVCF = x)
+  } validate { x =>
+    if (x.exists) success else failure("Input VCF not found")
+  } text "Input VCF file. Required."
+  opt[File]('O', "OutputFile") required () valueName "<vcf>" action { (x, c) =>
+    c.copy(outputVCF = x)
+  } validate { x =>
+    if (!x.getName.endsWith(".vcf") && (!x.getName.endsWith(".vcf.gz")) && (!x.getName.endsWith(
+      ".bcf")))
+      failure("Unsupported output file type")
+    else success
+  } text "Output VCF file. Required."
+
+  opt[String]('m', "mode") required () valueName "<mode>" action { (x, c) =>
+    c.copy(mode = x)
+  } validate { x =>
+    if (x == "explode") success
+    else if (x == "standard") success
```

Comment:
Is `expand` or `expanded` a good alternative for explode?

Reply:
I agree but since this is already in use we can't change this in this version